src/core/util/log.cc

```diff
@@ -131,6 +156,19 @@ void SetupLogging(const boost::program_options::variables_map& kovri_config)
   auto file_sink = boost::shared_ptr<text_file_sink>(
       std::make_unique<text_file_sink>(file_backend));
   // Set sink formatting
+  logging::formatter color_format = 
```

Comment:
Ideally, in a separate commit, we provide a config option to disable color log `--log-enable-color` which would default to true (so, enabled by default). If there is ample evidence that boost is smart enough to deal with non-color terminals, then nevermind.

Reply:
I think these color escape codes are not boost related.